recipes/itk/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "4.11.0" %}
+
+package:
+  name: itk
+  version: "{{version}}"
+
+source:
+  url: https://github.com/InsightSoftwareConsortium/ITK/archive/v{{version}}.tar.gz
+  sha256: a1857aa89f02efce3e091d102fbff7b2c53fe40c7f74580a47236bfd846c4e8b
+
+build:
+    number: 0
+    skip: True   # [win and py36]
+    features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
+
+requirements:
+  build:
+    - toolchain
```

Comment:
Note that all of the `vc` portions rely on selectors related to the version of `python` used. As such we need this to make sure those selectors work. Also if we don't have this listed, then we get multiple rebuilds of the package on Unix for each Python version, which we don't need.